app/backend/src/couchers/servicers/account.py

```diff
@@ -266,3 +266,16 @@ def VerifyPhone(self, request, context):
             user.phone_verification_attempts = 0
 
         return empty_pb2.Empty()
+
+    def DeleteAccount(self, request, context):
+        """
+        Cannot be used to delete any account, except for the user's own
+
+        Extensive checking should be confirmed by the frontend
+        """
+        with session_scope() as session:
+            user = session.execute(select(User).where(User.id == context.user_id)).scalar_one()
+            user.is_deleted = True
+            session.commit()
```

Comment:
The commit is automatic when the `with` context ends, but it might be good to do it explicitly like you do here anyway.